README.rst

```diff
@@ -62,33 +62,101 @@ It is suggested to install a pre-commit hook in order to apply Black before push
     $ pre-commit install
 
 
-Project Progres
+Project Progress
 ----------------
 
 View the project Kanban board `here <https://github.com/debrief/pepys-import/projects/3>`_
 
 Setup
 -----
 
-To prepare for running ensure these tools and packages are installed:
+To prepare for running ensure Python 3.7 or later are installed in your system.
+You can check your Python 3 version with the following command::
 
-* Python 3 - at least Python 3.6.4 or later
-* SQL Alchemy 1.3
+    $ python3 --version
 
-For Postgres support and unit tests these packages are also required:
+If you don't have Python 3.7+ in your system, please download it from `python.org <https://www.python.org/downloads/>`_
 
-* geoalchemy2
-* psycopg2
-* nose2 (0.9.1)
+Pip is also necessary to create virtual environment. If you don't have it in your system, please download it::
 
-Dependencies
-------------
+    $ sudo apt-get install python3-pip
+    $ pip3 install virtualenv
 
-Install the python dependencies with: :code:`pip install -r requirements.txt`
+It is possible to verify pip installation with the following command::
+
+    $ pip3 --version
+
+----------
+For Ubuntu
+----------
+
+Installing Spatialite
+*********************
+1. Easy Way
+"""""""""""
+
+Open a terminal and run the following command::
+
+    $ sudo apt install spatialite-bin
+
+2. Alternative Way (Compiling From Source)
+""""""""""""""""""""""""""""""""""""""""""
+
+On Debian-based distributions the following libraries are necessary to run SpatiaLite::
+
+    $ sudo apt install zlib1g-dev libfreexl1
+    $ sudo apt-get install sqlite3 libsqlite3-dev
+    $ sudo apt-get install- y libsqlite3-mod-spatialite
```

Comment:
@BarisSari  - typo above. Should be "-y" not "- y"

Reply:
Corrected:

https://github.com/debrief/pepys-import/pull/66/commits/d45910bf34f1b8ab38a58af0d88dc23f18c734d6